### DISCUSSION TITLE
7zip and yazi on Windows

### DISCUSSION THREAD

**Discussion by vbd** [sxyazi/yazi#1608] [1. Q&A]
Is there a guide how to configure and use yazi with 7-zip on Windows to compress and uncompress archives?
I would prefer not to use a Plugin.
I'm on Windows with yazi 0.3.

**Comment by sxyazi** [comment]
There's a plugin at https://github.com/KKV9/compress.yazi, although it's a plugin, it's very easy to use. 

If you don't want to use the plugin, you'd have to write your own command and execute it through [the `shell` command](https://yazi-rs.github.io/docs/configuration/keymap/#manager.shell), which should be much more difficult than using the plugin.

**Comment by vbd** [comment]
Ok. Will try it. Thanky you.
